seaworthy/containers/provided.py

```diff
@@ -116,6 +118,7 @@ def __init__(self,
                  name=DEFAULT_NAME,
                  image=DEFAULT_IMAGE,
                  wait_patterns=DEFAULT_WAIT_PATTERNS,
+                 create_kwargs=None,
```

Comment:
This seems unnecessary?

Reply:
We might want to pass a command to run or some default envvars or whatever in at construction time but then pass different things in when the container is created.

Reply:
Ah, think I misread the diff.